database/factories/HafasTripFactory.php

```diff
@@ -117,4 +113,35 @@ public function definition() {
             'polyline'    => $polylineHash,
         ];
     }
+
+    /**
+     * Configure the model factory.
+     *
+     * @return $this
+     */
+    public function configure() {
+        return $this->afterCreating(function(HafasTrip $hafasTrip) {
+            $stopOvers = json_decode($hafasTrip->stopovers);
+            $startTime = $hafasTrip->departure;
+            $endTime   = $hafasTrip->arrival;
+            $cnt       = count($stopOvers);
+            $interval  = (strtotime($endTime) - strtotime($startTime)) / $cnt;
+            $add       = $interval;
+
+            for ($i = 1; $i < $cnt; $i++) {
+                $stopOvers[$i]->arrival   = date('Y-m-d H:i:s', strtotime($startTime) + $add);
```

Comment:
$startTime is an Carbon object. So why do you put it in an strtotime to format it with date?

Just use $startTime->format('Y-m-d H:i:s').

Reply:
(Same with $endTime)

Reply:
Is it necessary to format these? Why dont just save the Carbon object in the $stopOvers? This should be formatted in ISO 8601 automatically on json_encode.

Reply:
Oh wait, you can do that? I thought, I should format it so there won't be any complications.

Reply:
Yeah, I think so, because the when you do the JSON Encode it will try to parse the toString-Method of an Object.
https://github.com/briannesbitt/Carbon/blob/db2173fbb9ebb97645c17ea23b1ec0c9be6abb0d/src/Carbon/Traits/Converter.php#L115